app/src/main/java/com/example/peterchu/watplanner/scheduler/ScheduleUtils.java

```diff
@@ -0,0 +1,38 @@
+package com.example.peterchu.watplanner.scheduler;
+
+import android.graphics.Color;
+
+import com.alamkanak.weekview.WeekViewEvent;
+import com.example.peterchu.watplanner.Models.Schedule.CourseComponent;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class ScheduleUtils {
+
+    private ScheduleUtils() {
+        // Util class
```

Comment:
Prevents instantiation of this class. Since this is a utils class with static methods only, we shouldn't allow this.